examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,202 @@
+                'microAmount': 10000000
+            }
+        }]
+        },
+        'adGroupAdRotationMode': 'OPTIMIZE'
+    }
+
+    adgroup_operations = [{
+        'operator': 'ADD',
+        'operand': ad_group
+    }]
+    results = ad_group_service.mutate(adgroup_operations)
+    createdAdgroup = results['value'][0]
+    print("Ad group with ID {} and name {} was created".\
+        format(createdAdgroup['id'],createdAdgroup['name']))
+    return createdAdgroup['id']
+
+
+def createTextAds(client, adGroupId):
+    ad_group_service = client.GetService('AdGroupAdService', 'v201809')
+    operations = []
+    for i in range(NUMBER_OF_ADS):
+        operation =  {
+            'xsi_type': 'AdGroupAd',
+            'adGroupId': adGroupId,
+            # Additional properties (non-required).
+            'status':  'PAUSED',
+            'ad': {
+                'xsi_type': 'ExpandedTextAd',
+                'headlinePart1': ('Cruise #%s to Mars'
+                            % str(uuid.uuid4())[:8]),
+                'headlinePart2': 'Best Space Cruise Line',
+                'headlinePart3': 'For Your Loved Ones',
+                'description': 'Buy your tickets now!',
+                'description2': 'Discount ends soon',
+                'finalUrls': ['http://www.example.com/']
+            }
+        }
+        adgroup_operations = {
+            'operator': 'ADD',
+            'operand': operation
+         }
+        operations.append(adgroup_operations)
+    results = ad_group_service.mutate(operations)
+    for result in results['value']:
+        print('Expanded text ad with ID {} and headline {}-{} {} was created'\
+              .format(result['ad']['id'],result['ad']['headlinePart1'],\
+               result['ad']['headlinePart2'],result['ad']['headlinePart3']))
+
+
+def createKeywords(client, adGroupId, keywordsToAdd):
+    AdGroupCriterionService = client.GetService("AdGroupCriterionService", \
+                                              'v201809')
+    operations = []
+    for keyword in KEYWORDS_TO_ADD:
+        operation = {
+        'xsi_type': 'BiddableAdGroupCriterion',
+        'adGroupId': adGroupId,
+        'criterion': {
+            'xsi_type' : 'Keyword',
+            'text': keyword,
+            'matchType' : 'BROAD'
+        },
+        'userStatus': 'PAUSED',
+        'finalUrls' : ['http://www.example.com/mars/cruise/?kw=%s'% \
+                       urllib.parse.quote(keyword)]
+        }
+        create_keyword = {
+            'operator': 'ADD',
+            'operand': operation
+        }
+        operations.append(create_keyword)
+    results = AdGroupCriterionService.mutate(operations)
+    for result in results['value']:
+        print("Keyword with ad group ID {}, keyword ID {}, text {} and match"
+              "type {} was created".format(result['adGroupId'], \
+               result['criterion']['id'],result['criterion']['text'],\
+               result['criterion']['matchType']))
```

Comment:
Similar feedback as above - no need to use `\` for continuation, add spaces after commas.